service/src/main/java/uk/gov/hmcts/reform/fpl/service/DateFormatterService.java

```diff
@@ -11,7 +11,10 @@
 @Service
 public class DateFormatterService {
     public String formatLocalDateToString(LocalDate date, FormatStyle style) {
-        return date.format(DateTimeFormatter.ofLocalizedDate(style).localizedBy(Locale.UK));
+        if (date != null) {
```

Comment:
why the null check here as the caller will still do a null check?

Reply:
fixed